src/bucket/BucketList.cpp

```diff
@@ -179,10 +180,12 @@ BucketLevel::snap()
 {
     mSnap = mCurr;
     mCurr = std::make_shared<Bucket>();
-    // CLOG(DEBUG, "Bucket") << "level " << mLevel << " set mSnap to "
-    //            << mSnap->getEntries().size() << " elements";
-    // CLOG(DEBUG, "Bucket") << "level " << mLevel << " reset mCurr to "
-    //            << mCurr->getEntries().size() << " elements";
+    // CLOG_DEBUG(Bucket, "level {}{}{} elements", mLevel, " set mSnap
```

Comment:
same thing: commented out logging statements are not useful. remove or TRACE

Reply:
fixed.